src/components/pages/areas-manage/AreasManage.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { NavLink } from 'react-router-dom';
```

Comment:
`NavLinks` are meant to be used in a nav bar. There's `Link` for the other use cases.